test/tstDetailsCrsGraphWrapperImpl.cpp

```diff
@@ -61,15 +61,15 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(query_impl, DeviceType, ARBORX_DEVICE_TYPES)
 
   int const buffer_size = 2 * (n + 1);
 
-  ArborX::reallocWithoutInitializing(offset, n + 1);
+  Kokkos::realloc(offset, n + 1);
```

Comment:
This seems off, doesn't realloc initialize? Or is it simply due to this being in a test?
Similar in all other tests.

Reply:
Because it is ok in unit tests